multichain-testing/scripts/ymax-tool.ts

```diff
@@ -99,14 +102,35 @@ const openPosition = async (
   );
   trace('tx', actual);
 
-  const status = await walletKit.pollOffer(
-    address,
-    action.offer.id,
-    before.header.height,
-  );
-  trace('status', status);
-  if ('error' in status) throw Error(status.error);
-  return status;
+  if (!skipPoll) {
```

Comment:
Consider avoiding another level of indentation for the normal polling case by using an early return:

```suggestion
  if (skipPoll) {
    ...
    return;
  }
  ...polling code...
```

Reply:
Good catch. Done.